tests/testsuite/workspaces.rs

```diff
@@ -2658,22 +2658,25 @@ fn warn_package_togother_with_workspace() {
     // baz is present in the workspace and it is a real existing package.
     p.cargo("check --package baz --workspace")
         .with_stderr_data(str![[r#"
-[CHECKING] foo v0.1.0 ([ROOT]/foo)
+[WARNING] ignoring `--package` as the `--workspace` or `--all` flag is set
```

Comment:
The issue wasn't about redundant `--package` flags but about non-existent packages.  *If* we decide to warn on redundant flags, then we can simplify the design of this but that is something that needs to be figured out first.

Reply:
> The issue wasn't about redundant --package flags but about non-existent package.

My PR was deviated from the original question. I tried to add warning for non-existent package.